js/table/tableview.ts

```diff
@@ -1524,10 +1535,22 @@ export class TableView {
                 break;
             }
             default:
-                caption = ""
+                caption = "";
         }
         this.roundNotification(caption);
     }
+    /**
+     * Update round notification timer
+     */
+    public updateRoundNotificationTimer() {
```

Comment:
This method and all dealing with timeout should be part of tableManager and outside of TableView responsibility. The whole behavior for showing notifications could be later extracted out of tableManager and put in the separate class which will do only single thing - show notifications